data_structures/src/chain.rs

```diff
@@ -1584,14 +1584,21 @@ impl Hashable for Bn256Signature {
 /// Retrieval type
 #[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize, Hash)]
 pub enum RADType {
+    /// Unknown
+    #[serde(rename = "Unknown")]
+    Unknown,
     /// HTTP GET request
     #[serde(rename = "HTTP-GET")]
     HttpGet,
+    /// Random number generation
+    #[serde(rename = "RNG")]
+    Rng,
 }
 
 impl Default for RADType {
     fn default() -> Self {
-        RADType::HttpGet
+        // TODO: Use RadType::HttpGet after WIP-0019 activation
+        RADType::Unknown
```

Comment:
I think `RADType::Unknown` is a good default value, the problem is that this is used in tests so any tests that want to create a "default HttpGet request" can use a helper function.
```suggestion
        RADType::Unknown
```

Reply:
I think we could do a refactor in tests after WIP19 activation